core/cloudflow-it/swiss-knife/project/cloudflow-plugins.sbt

```diff
@@ -1,5 +1,5 @@
 val latestVersion = {
-  sys.env.get("CLOUDFLOW_VERSION").fold(
+  sys.env.get("CLOUDFLOW_VERSION").filter(_.nonEmpty).fold(
```

Comment:
have you done a global search and replace for this?

Reply:
Yes. :-)